components/SearchForm/index.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import StyledForm from "../Form/StyledForm";
+import StyledList from "../List/StyledList";
+import { atom, useAtom } from "jotai";
+import globalTranslations from "@/public/store";
+import SVGIcon from "../Icons/SVGIcon";
+import styled from "styled-components";
+import Link from "next/link";
+import Divider from "../Divider";
+import RefreshButton from "../Buttons/RefreshButton";
+import StyledMessage from "../List/Message/StyledMessage";
+
+export default function SearchForm({ selectedLanguage }) {
+  const [translationList] = useAtom(globalTranslations);
+  const [searchInput, setSearchInput] = useState("");
+
+  const searchResults = translationList
+    .filter((translation) => {
+      if (selectedLanguage) {
+        return translation.language === selectedLanguage;
+      }
+      return true;
+    })
+    .filter((translation) => translation.word.indexOf(searchInput) === 0);
+
+  return (
+    <>
+      <StyledForm type="search">
+        <StyledInputWrapper>
+          <label htmlFor="searchValue"></label>
+          <StyledInput
+            id="searchValue"
+            name="searchValue"
+            placeholder="🔍"
+            value={searchInput}
+            onChange={(event) => setSearchInput(event.target.value)}
+          />
+        </StyledInputWrapper>
+        <RefreshButton onClick={() => setSearchInput("")} />
+      </StyledForm>
+      {searchResults.length === 0 && searchInput.length > 0 ? (
+        <StyledSection>
+          <StyledMessage>Not found</StyledMessage>
+        </StyledSection>
+      ) : null}
+      {searchInput.length > 0 && (
+        <StyledList>
+          {searchResults
+            .filter((translation) => translation.word.includes(searchInput))
+            .map((translation) => {
+              return (
+                <StyledSection
+                  key={translation.word}
+                  onClick={() => {
+                    setSearchInput(translation.word);
+                  }}
+                >
+                  <StyledMessage>
+                    {translation.word} -{" "}
+                    {!selectedLanguage ? (
+                      <small>({translation.language}) - </small>
+                    ) : (
+                      ""
+                    )}
```

Comment:
I think you always should display the language no matter if I selected a language or not